server/odc-service/src/main/java/com/oceanbase/odc/service/resultset/ResultSetExportTask.java

```diff
@@ -266,6 +276,21 @@ private void setColumnConfig(DataTransferConfig config, ResultSetExportTaskParam
         }
     }
 
+    private String getTimeoutSettingStatement(DialectType dialectType) {
```

Comment:
you may use this for timeout setting
ConsoleTimeoutInitializer

Reply:
thanks, updated.